packages/webdriverio/tests/scripts/newWindow.test.js

```diff
@@ -0,0 +1,11 @@
+/* global window */
+
+import newWindow from '../../src/scripts/newWindow'
+
+describe('newWindow script', () => {
+    it('should check if elem is active', () => {
+        window.open = jest.fn()
+        newWindow('foo', 'bar', 'loo')
+        expect(window.open.mock.calls).toHaveLength(1)
```

Comment:
`newWindow` takes `url, windowName, windowFeatures`. Let's add one more `expect` to be sure that it has been called with correct args.